models/parser/ExampleXlsParser.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * @author Patryk Radziszewski <pradziszewski@netis.pl>
+ * @link http://netis.pl/
+ * @copyright Copyright (c) 2015 Netis Sp. z o. o.
+ * @TODO trace error messages
+ */
+
+namespace nineinchnick\sync\models\parser;
+
+use netis\assortment\models\search\Product;
+use netis\contractors\models\Contractor;
+use netis\orders\models\OrderStatus;
+use nineinchnick\sync\models\parser\BaseXlsParser;
+use yii\base\InvalidCallException;
+use netis\orders\models\Order;
+use netis\orders\models\OrderItem;
+
+class ExampleXlsParser extends BaseXlsParser
+{
+    /**
+     * Either uploads a prepared file or downloads next file from a remote service.
+     * Returns bool false if no files are available.
+     * @param File $file
+     * @return File|bool bool true when uploading, bool false when downloading and no files are available
+     */
+    public function transfer($file = null)
+    {
+        $file->sent_on = date('Y-m-d H:i:s');
+        $file->save(false);
+        return true;
+    }
+
+    /**
+     * Parses the files contents.
+     * @param $file
+     * @return bool
+     * @throws Exception
+     */
+    public function process($file)
+    {
+        $configuration = $file->parserConfiguration;
```

Comment:
@pradziszewski123 I know this is just an example, but again, remove project specific code.